metadata_backend/api/operators.py

```diff
@@ -488,6 +495,21 @@ def format_date(key: str, doc: Dict) -> Dict:
             doc = format_date("publishDate", doc)
         return doc
 
+    async def check_exists(self, schema_type: str, accession_id: str) -> None:
```

Comment:
`Operator(BaseOperator)` and `XMLOperator(BaseOperator)` both use `check_exists()` and they both inherit `BaseOperator`, so can `check_exists()` be moved to `BaseOperator` class, so that the function is not duplicated?

Reply:
it is an a `ABC` https://docs.python.org/3/library/abc.html and the method needs to be implemented in the subclass

Reply:
but it is a good point I can reimplement it a bit better as it is used in other places as well like user and `folder`.

Reply:
previously when i had it as a reusable function it complained now it does not :shrug: